app/views/order_management/order_details/_print_details.html.haml

```diff
@@ -0,0 +1,35 @@
+%div.manage_order_detail
+  .banner-list
+    .row
+      %dl
+        %dt= @order_detail.class.human_attribute_name(:account)
+        %dd= @order_detail.account
+    = render_view_hook "after_account_print", order_detail: @order_detail
+    .row
+      = banner_label @order_detail, :order_status
+      = banner_date_label @order_detail, :reconciled_at if @order_detail.reconciled_at
+    .row
+      = banner_date_label @order_detail.reservation, :reserve_start_at, OrderDetail.human_attribute_name(:reserve_start_date).titleize
```

Comment:
We need a conditional in here for if there is a reservation include the times, but if it's not a reservation, then show the quantity.